### ISSUE TITLE
Add hook pre-commit to run eslint

### ISSUE THREAD

**Issue by mazipan** [mazipan/hello-open-source#6]
After #3 and #5 already solved, we need to force running eslint every before committing file.

Use `husky` for add git hook pre-commit.
Sample of this hook, you can see here https://github.com/mazipan/awesome-sde-id-medium/blob/master/package.json

**Comment by ninariccimarie** [comment]
hi @mazipan I'd like to take on this issue 🙏

**Comment by ninariccimarie** [comment]
also since, eslint is not yet installed, I'll install it and do the script to run the eslint.
will you have me do the eslint file on a separate PR though? I'd like to know what configurations you want me to select/do.

**Comment by mazipan** [comment]
Standard.js will be good I think
